src/main/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -171,6 +172,7 @@ class StockMovement implements Validateable{
             dateCreated         : dateCreated?.format("MM/dd/yyyy"),
             dateShipped         : dateShipped?.format("MM/dd/yyyy HH:mm XXX"),
             expectedDeliveryDate: expectedDeliveryDate?.format("MM/dd/yyyy HH:mm XXX"),
+            desiredDeliveryDate : requisition?.desiredDeliveryDate?.format("MM/dd/yyyy"),
```

Comment:
why requisition and not the desiredDeliveryDate field you added?

Reply:
I think we wanted to get rid of formatting a date on the backend and let the frontend handle the eventual formatting.
How about removing `.format` here and formatting it with Moment on the frontend?